src/QuantumOpticsBase.jl

```diff
@@ -80,5 +80,6 @@ include("metrics.jl")
 include("spinors.jl")
 include("phasespace.jl")
 include("printing.jl")
+#include("operators_lazyaritmetic.jl")
```

Comment:
this probably should be deleted before merging

Reply:
Oh yeah... Done that now ;)